add-on/chrome/bootstrap.js

```diff
@@ -607,6 +660,10 @@ var WindowListener = {
             if (event.detail.previousTab) {
               wasVisible = this._hideBrowserSharingInfoBar(event.detail.previousTab.linkedBrowser);
             }
+            // Remove the cursor from the previous tab.
+            if (event.detail.previousTab) {
+              this.removeRemoteCursor(event.detail.previousTab.linkedBrowser);
```

Comment:
The `if` check is the same, so just move the `removeRemoteCursor` call call inside and update the comment.